src/index.js

```diff
@@ -32,35 +36,146 @@ class Book {
 
 const newBook = new Book();
 
-addBtn.addEventListener('click', (e) => {
-  e.preventDefault();
-  const titleText = title.value;
-  const authorText = author.value;
-  newBook.add(titleText, authorText);
-  document.location.reload();
-});
+const listNavMenu = () => {
+  const allBooks = document.createElement('h1');
+  allBooks.innerText = 'All awesome books';
+  listbooks.forEach((book) => {
+    const bookCard = document.createElement('div');
+    bookCard.classList.add('bookCard');
+    const bookTitle = document.createElement('p');
+    bookTitle.classList.add('bookTitle');
+    bookTitle.innerText = `"${book.title}" by ${book.author}`;
+    const bookAuthor = document.createElement('p');
+    bookAuthor.classList.add('bookAuthor');
+    bookAuthor.innerText = book.author;
+    const removeBtn = document.createElement('button');
+    removeBtn.classList.add('remove');
```

Comment:
- [ ] Great job 👏🏻 However, when any of the links are clicked, the main area changes to display only one of the three content sections. kindly make sure you address this ✍🏻  You could add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) add and remove methods. 
- Here is the screen video of you app:

[Screencast from 08-04-2022 03:00:51 AM.webm](https://user-images.githubusercontent.com/79058364/182741229-147f265b-8023-4c16-b1ee-5aa11a4ffb19.webm)

Reply:
@BenMukebo Let me check it, everything was working fine.

Reply:
@HabibPayenda Please check well !!